codec-http2/src/main/java/io/netty/handler/codec/http2/Http2MultiplexCodec.java

```diff
@@ -772,6 +772,19 @@ void fireChildReadComplete() {
             }
         }
 
+        void fireChildExceptionCaught(Throwable cause) {
```

Comment:
I don't like this method, or at least its name. It is not obvious it has side effects other than propagating the exception. Either call wrap explicitly or maybe make this `wrapAndFireChildExceptionCaught` or similar.

Reply:
Sure... just used this naming to be consistent with other names

Reply:
I kept it like this now as there is nothing special going on here anymore (the exception wrapping only happens for outbound).

Reply:
My vote is to remove the method for now. We can add it back later if necessary.

Reply:
sure its not doing anything except delegating anyway atm